Sources/AssetGen/Infrastructure/SwiftLocalizeGenerator/Processor/PlaceholderExtractor.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+struct PlaceholderExtractor {
+  func extractPlaceholders(from translation: String) -> [Placeholder] {
+    let pattern = "(%([0-9]*\\$)?(\\d*\\$)?[@diufFeEgGxXoOcsaAbBhH]+)"
+    guard let regex = try? NSRegularExpression(pattern: pattern, options: []) else { return [] }
```

Comment:
Could we go with the new [Regex](https://developer.apple.com/documentation/swift/regex) type instead?

Reply:
Actually we are supporting macOS 12+, and Regex is 13+

Reply:
Since most devs will be using this, don't see the reason for supporting macOS 12. And we are getting 14 this year as well. What are your thoughts?

Reply:
makes sense 👍 , let me clean that up